server/belga/search_providers.py

```diff
@@ -509,10 +509,27 @@ def format_list_item(self, data):
             "sign_off": self.get_sign_off(data.get("authors")),
             "authors": self.get_authors(data.get("authors")),
             "subject": self.get_subjects(data),
+            "renditions": self.get_renditions(data)
+            if data["assetType"] == "Picture"
```

Comment:
not sure if the `assetType` is always set, given that we're using `data.get('assetType')` before

Reply:
here we check the `assetType` which we get from the api response, and I checked several items we got `assetType` in every item.

Reply:
yep but when detecting the type we use it with `.get`, so either we don't need it there or we should use `.get` here as well to be consistent